index.html

```diff
@@ -0,0 +1,1209 @@
+
+                  <p>WAC has the property of being recursive with respect to container hierarchy, meaning that a member resource inherits Authorizations from the closest container resource (heading toward the root container).</p>
+
+                  <p>To determine the <em>effective ACL resource</em> of a resource, perform the following steps. Returns <a href="https://infra.spec.whatwg.org/#strings">string</a> (the URI of an ACL Resource).</p>
+
+                  <ol>
+                    <li>Let <var>resource</var> be the <a href="#resource">resource</a>.</li>
+                    <li>Let <var>aclResource</var> be the <a href="#acl-resource">ACL resource</a> of <var>resource</var>.</li>
+                    <li>If <var>resource</var> has an associated <var>aclResource</var> with a representation, return <var>aclResource</var>.</li>
+                    <li>Otherwise, repeat the steps using the <a href="#container-resource">container resource</a> of <var>resource</var>.</li>
+                  </ol>
+
+                  <div class="note" id="effective-acl-resource-alternatives" inlist="" rel="schema:hasPart" resource="#effective-acl-resource-alternatives">
+                    <h4 property="schema:name"><span>Note</span>: Effective ACL Resource Alternatives</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>This specification does not constrain alternative methods to determine the effective ACL resource of a resource.</p>
+                    </div>
+                  </div>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/specification/issues/55" rel="cito:citesAsSourceDocument">issues/55</a>, <a href="https://github.com/solid/specification/issues/106" rel="cito:citesAsSourceDocument">issues/106</a>, <a href="https://github.com/solid/specification/issues/130" rel="cito:citesAsSourceDocument">issues/130</a>, <a href="https://github.com/solid/specification/issues/257" rel="cito:citesAsSourceDocument">issues/257</a>, <a href="https://github.com/solid/specification/issues/251" rel="cito:citesAsSourceDocument">issues/251</a></p>
+                </div>
+              </section>
+
+              <section id="authorization-conformance" inlist="" rel="schema:hasPart" resource="#authorization-conformance">
+                <h3 property="schema:name">Authorization Conformance</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>An applicable Authorization has the following properties:</p>
+
+                  <ul>
+                    <li>At least one <code>rdf:type</code> property whose object is <code>acl:Authorization</code>.</li>
```

Comment:
Why not exacly one?

Reply:
Because it is RDF.. and including 50 million of those wouldn't be wrong or break anything - as far as I know :)